tests/test/test_translation.py

```diff
@@ -0,0 +1,157 @@
+import json
+
+from bs4 import BeautifulSoup as bs4
+from django.test import override_settings
+from django.urls import reverse
+from django.utils import translation
+from wagtail.models import Locale, Page
+from wagtail.test.utils import TestCase, WagtailTestUtils
+
+from wagtail_footnotes.models import Footnote
+
+from ..models import TestPageStreamField
+
+
+@override_settings(
+    LANGUAGES=[
+        ("en", "English"),
+        ("fr", "French"),
+        ("de", "German"),
+    ],
+    WAGTAIL_CONTENT_LANGUAGES=[
+        ("en", "English"),
+        ("fr", "French"),
+        ("de", "German"),
+    ],
+)
+class TestSubmitPageTranslationView(WagtailTestUtils, TestCase):
+    def setUp(self):
```

Comment:
question: why not use
```python
@classmethod
def setUpTestData(cls)
   ...
```

which is only run once for the test class, rather than each time when a test case is se tup?

Reply:
Done